package.json

```diff
@@ -23,6 +24,20 @@
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
+    "bootstrap": "^3.3.7",
```

Comment:
Seems like dependencies and devDependencies are mixed up.
Everything that would be needed to build the app on production should listed as "dependencies" node. Things that are not for production like mocha should be listed as "devDependencies".
https://github.com/webpack/webpack/issues/520